digital_image_processing/dithering/burkes.py

```diff
@@ -39,20 +39,24 @@ def __init__(self, input_img, threshold: int):
     def get_greyscale(cls, blue: int, green: int, red: int) -> float:
         """
         >>> Burkes.get_greyscale(3, 4, 5)
-        3.753
+        4.185
+        >>> Burkes.get_greyscale(0, 0, 0)
+        0.0
+        >>> Burkes.get_greyscale(255, 255, 255)
+        255.0
         """
-        return 0.114 * blue + 0.587 * green + 0.2126 * red
+        return 0.114 * blue + 0.587 * green + 0.299 * red
```

Comment:
Could you add a source for these numbers so that we can avoid mistakes like this in the future?

Reply:
Good idea, done :)